sdk/agent/config/config_helpers.go

```diff
@@ -56,6 +57,8 @@ func GetKnownExtensions() []string {
 		AdvancedMetricsExtensionPlugin,
 		NginxAppProtectExtensionPlugin,
 		NginxAppProtectMonitoringExtensionPlugin,
+		// TODO : Enable php-fpm metric extension functionality
```

Comment:
when do we expect to resolve this TODO?

Reply:
This would be tackled, once it's feature complete in dev env. Strategy of handling it during our stage test is TBD